lib/europeana/blacklight/search_builder.rb

```diff
@@ -99,8 +102,26 @@ def add_facet_qf_to_api(api_parameters)
         end
       end
 
-      # bizarrely, reusability is a distinct API param, even though it
-      # is returned with the facets in a search response
+      ##
+      # Filter results by a query facet
+      def add_query_facet_to_api(api_parameters)
```

Comment:
Unused method argument - `api_parameters`. If it's necessary, use `_` or `_api_parameters` as an argument name to indicate that it won't be used. You can also write as `add_query_facet_to_api(*)` if you want the method to accept any arguments but don't care about them.<br>Assignment Branch Condition size for add_query_facet_to_api is too high. [25.4/15]